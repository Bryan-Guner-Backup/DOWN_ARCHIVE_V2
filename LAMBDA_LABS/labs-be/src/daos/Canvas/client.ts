import { Request, Response } from "express";
import {
  BaseClient,
  AuthTypes,
  ClientResponse,
  ClientArrayResponse,
} from "@shared/BaseClient";

export interface ICanvasClient<T> {
  get: (path: string) => ClientResponse<T> | ClientArrayResponse<T>;
}

class CanvasClient<T> extends BaseClient<T> implements ICanvasClient<T> {
  constructor() {
    const opts = {
      token: process.env.CANVAS_ACCESS_TOKEN,
      baseUrl: "https://lambdaschool.instructure.com/api/v1/",
      authType: AuthTypes.JWT,
    };

    super(opts);
  }
}

export default CanvasClient;
