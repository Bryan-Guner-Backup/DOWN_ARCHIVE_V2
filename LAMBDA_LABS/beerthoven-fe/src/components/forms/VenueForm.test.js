import "@testing-library/jest-dom/extend-expect";
import { cleanup, fireEvent, render, wait } from "@testing-library/react";
import { createMemoryHistory } from "history";
import React from "react";
import { ApolloProvider } from "react-apollo";
import { Router } from "react-router";
import venue from "../../test-data/venue";
import client from "../graphql/client";
import VenueForm from "./VenueForm";

//to see if mock info is able to be retrieved
const onSubmit = jest.fn();

describe(`Event Form Component`, () => {
  beforeEach(cleanup);
  afterEach(cleanup);
  it(`renders a component that uses withRouter`, async () => {
    const history = createMemoryHistory();
    const route = "/some-route";
    history.push(route);
    const { container } = render(
      <ApolloProvider client={client}>
        <Router history={history}>
          <VenueForm />
        </Router>
      </ApolloProvider>
    );

    expect(container.firstChild).toMatchSnapshot();
  });

  it(`should fetch the required data`, async () => {
    const history = await createMemoryHistory();
    const { getByTestId, getByLabelText, getByText } = await render(
      <ApolloProvider client={client}>
        <Router history={history}>
          <VenueForm venue={venue} />
        </Router>
      </ApolloProvider>
    );

    const nameNode = await getByLabelText("Name");
    const venueTypeNode = await getByLabelText("Venue Type");
    const addressNode = await getByLabelText("Address");
    const cityNode = await getByLabelText("City");
    const stateNode = await getByLabelText("State");
    const zipCodeNode = await getByLabelText("Zip code");
    const maxCapacityNode = await getByLabelText("Max Capacity");
    const minimumIncomeNode = await getByLabelText("Min Income");
    const depositAmountNode = await getByLabelText("Deposit Amount");
    const alcoholBeerServedNode = await getByTestId("smoking_allowed");
    const alcoholWineServedNode = await getByTestId("alcohol_wine_provided");
    const smokingAllowedNode = await getByTestId("smoking_allowed");
    const under21Node = await getByTestId("under21_allowed");
    const under18Node = await getByTestId("under18_allowed");
    const wheelchairNode = await getByTestId("wheelchair_accessible");
    const alcoholSpiritsServedNode = await getByTestId(
      "alcohol_spirits_provided"
    );
    const foodServedNode = await getByTestId("food_served");
    const tabCCertifiedNode = await getByTestId("tabc_certified");
    const indoorVenueNode = await getByTestId("indoor_venue");
    const outdoorVenueNode = await getByTestId("outdoor_venue");
    const parkingLotAvailableNode = await getByTestId("parking_lot_available")
      .children[0];

    const submitBtn = await getByText("Submit");

    wait(() => {
      fireEvent.change(nameNode, { target: { value: venue.name } });
      fireEvent.change(venueTypeNode, { target: { value: venue.venue_type } });
      fireEvent.change(addressNode, { target: { value: venue.address } });
      fireEvent.change(cityNode, { target: { value: venue.city } });
      fireEvent.change(stateNode, { target: { value: venue.state } });
      fireEvent.change(zipCodeNode, { target: { value: venue.zip } });
      fireEvent.change(maxCapacityNode, {
        target: { value: venue.max_capacity },
      });
      fireEvent.change(minimumIncomeNode, {
        target: { value: venue.min_income },
      });
      fireEvent.change(depositAmountNode, {
        target: { value: venue.deposit_amount },
      });
      fireEvent.change(alcoholBeerServedNode, {
        target: { value: venue.alcohol_beer_provided },
      });
      fireEvent.change(alcoholWineServedNode, {
        target: { value: venue.alcohol_wine_provided },
      });

      fireEvent.submit(submitBtn); //test is red when fireEvent.submit()

      fireEvent.change(parkingLotAvailableNode, {
        target: { value: venue.parking_lot_available },
      });
      fireEvent.change(smokingAllowedNode, {
        target: { value: venue.smoking_allowed },
      });
      fireEvent.change(under21Node, {
        target: { value: venue.under21_allowed },
      });
      fireEvent.change(under18Node, {
        target: { value: venue.under18_allowed },
      });
      fireEvent.change(wheelchairNode, {
        target: { value: venue.wheelchair_accessible },
      });
      fireEvent.change(alcoholSpiritsServedNode, {
        target: { value: venue.alcohol_spirits_served },
      });
      fireEvent.change(foodServedNode, {
        target: { value: venue.food_served },
      });
      fireEvent.change(tabCCertifiedNode, {
        target: { value: venue.smoking_allowed },
      });
      fireEvent.change(indoorVenueNode, {
        target: { value: venue.indoor_venue },
      });
      fireEvent.change(outdoorVenueNode, {
        target: { value: venue.outdoor_venue },
      });
    });

    await onSubmit(venue);
    expect(onSubmit).toHaveBeenCalledTimes(1);
    expect(onSubmit).toHaveBeenCalled();
    expect(onSubmit).toHaveBeenCalled();
  });

  it("Data matches required mock data as expected", async () => {
    expect(onSubmit).toHaveBeenCalledWith({
      name: "Biggest Venue",
      venue_type: "Large",
      address: "1200 lane",
      city: "The big city",
      state: "NY",
      zip: "14552",
      max_capacity: 1204,
      min_income: 25000,
      deposit_amount: 24000,
      smoking_allowed: false,
      under21_allowed: true,
      under18_allowed: false,
      wheelchair_accessible: false,
      alcohol_beer_provided: true,
      alcohol_wine_provided: true,
      alcohol_spirits_provided: true,
      food_served: true,
      max_decibel: 2000,
      opening_time: "7am",
      closing_time: "9pm",
      dance_floor_size: "Pretty Big",
      indoor_venue: false,
      outdoor_venue: true,
      parking_lot_available: true,
      parking_max_capacity: 1200,
      tabc_certified: true,
    });
  });
});
