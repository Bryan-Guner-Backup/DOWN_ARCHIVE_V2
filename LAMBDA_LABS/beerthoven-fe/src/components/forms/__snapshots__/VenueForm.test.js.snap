// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event Form Component renders a component that uses withRouter 1`] = `
<form
  class="ant-form ant-form-vertical"
>
  <h1>
    Add New Venue
  </h1>
  <div
    class="ant-row"
    style="margin: -8px -8px 8px -8px;"
  >
    <div
      class="ant-col ant-col-12"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="name"
            title="Name"
          >
            Name
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="name"
                placeholder="Enter venue name"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="ant-row"
    style="margin: -8px -8px 8px -8px;"
  >
    <div
      class="ant-col ant-col-12"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="venue_type"
            title="Venue Type"
          >
            Venue Type
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="venue_type"
                placeholder="Venue Type"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="ant-row"
    style="margin: -8px -8px 8px -8px;"
  >
    <div
      class="ant-col"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="address"
            title="Address"
          >
            Address
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="address"
                placeholder="Enter street address"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="ant-row"
    style="margin: -8px -8px 8px -8px;"
  >
    <div
      class="ant-col ant-col-12"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="city"
            title="City"
          >
            City
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="city"
                placeholder="Enter city"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="ant-row"
    style="margin: -8px -8px 8px -8px;"
  >
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="state"
            title="State"
          >
            State
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="state"
                placeholder="Enter state"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="zip"
            title="Zip code"
          >
            Zip code
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="zip"
                placeholder="Enter zip code"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-12"
      style="padding: 8px 8px 8px 8px;"
    />
  </div>
  <div
    class="ant-row"
    style="margin: -8px -8px 8px -8px;"
  >
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
        placeholder="Max Capacity"
        type="number"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            for="max_capacity"
            title="Max Capacity"
          >
            Max Capacity
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="max_capacity"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            for="min_income"
            title="Min Income"
          >
            Min Income
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="min_income"
                placeholder="Minimum Income"
                type="number"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            for="deposit_amount"
            title="Deposit Amount"
          >
            Deposit Amount
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="deposit_amount"
                placeholder="Deposit Amount"
                type="number"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="ant-row"
    style="margin: -8px -8px 8px -8px;"
  >
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="smoking_allowed"
            title="Smoking Allowed"
          >
            Smoking Allowed
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="smoking_allowed"
                      name="smoking_allowed"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="smoking_allowed"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="under21_allowed"
            title="Under 21 Allowed"
          >
            Under 21 Allowed
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="under21_allowed"
                      name="under21_allowed"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="under21_allowed"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="under18_allowed"
            title="Under 18 Allowed"
          >
            Under 18 Allowed
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="under18_allowed"
                      name="under18_allowed"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="under18_allowed"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="wheelchair_accessible"
            title="Wheelchair Accessible"
          >
            Wheelchair Accessible
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="wheelchair_accessible"
                      name="wheelchair_accessible"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="wheelchair_accessible"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="alcohol_beer_provided"
            title="Beer Served"
          >
            Beer Served
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="alcohol_beer_provided"
                      name="alcohol_beer_provided"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="alcohol_beer_provided"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="alcohol_wine_provided"
            title="Wine Served"
          >
            Wine Served
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="alcohol_wine_provided"
                      name="alcohol_wine_provided"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="alcohol_wine_provided"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="alcohol_spirits_provided"
            title="Alcohol Spirits Served"
          >
            Alcohol Spirits Served
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="alcohol_spirits_provided"
                      name="alcohol_spirits_provided"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="alcohol_spirits_provided"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="food_served"
            title="Food Served"
          >
            Food Served
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="food_served"
                      name="food_served"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="food_served"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            for="max_decibel"
            title="Max Decibel"
          >
            Max Decibel
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="max_decibel"
                placeholder="Max Decibel"
                type="number"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            for="opening_time"
            title="Opening Time"
          >
            Opening Time
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="opening_time"
                placeholder="Opening Time"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            for="closing_time"
            title="Closing Time"
          >
            Closing Time
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="closing_time"
                placeholder="Closing Time"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            for="dance_floor_size"
            title="Dance Floor Size"
          >
            Dance Floor Size
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="dance_floor_size"
                placeholder="Dance Floor Size"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="indoor_venue"
            title="Indoor Venue"
          >
            Indoor Venue
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="indoor_venue"
                      name="indoor_venue"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="indoor_venue"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="outdoor_venue"
            title="Outdoor Venue"
          >
            Outdoor Venue
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="outdoor_venue"
                      name="outdoor_venue"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="outdoor_venue"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-col ant-col-6"
      style="padding: 8px 8px 8px 8px;"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="parking_lot_available"
            title="Parking Lot Available"
          >
            Parking Lot Available
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="parking_lot_available"
                      name="parking_lot_available"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="parking_lot_available"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="ant-row"
  >
    <div
      class="ant-col ant-col-6"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            for="parking_max_capacity"
            title="Parking Max Capacity"
          >
            Parking Max Capacity
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input"
                id="parking_max_capacity"
                placeholder="Parking Max Capacity"
                type="number"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="ant-row"
  >
    <div
      class="ant-col ant-col-6"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="tabc_certified"
            title="TabC Certified"
          >
            TabC Certified
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      data-testid="tabc_certified"
                      name="tabc_certified"
                      type="radio"
                      value="false"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    No
                  </span>
                </label>
                <label
                  class="ant-radio-wrapper"
                >
                  <span
                    class="ant-radio"
                  >
                    <input
                      class="ant-radio-input"
                      name="tabc_certified"
                      type="radio"
                      value="true"
                    />
                    <span
                      class="ant-radio-inner"
                    />
                  </span>
                  <span>
                    Yes
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="ant-row ant-form-item"
  >
    <div
      class="ant-col ant-form-item-control"
    >
      <div
        class="ant-form-item-control-input"
      >
        <div
          class="ant-form-item-control-input-content"
        >
          <button
            class="ant-btn ant-btn-primary"
            type="submit"
          >
            <span>
              Submit
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
</form>
`;
